x-pack/solutions/search/plugins/enterprise_search/public/applications/applications/components/search_applications/search_application_indices_flyout.tsx

```diff
@@ -44,7 +42,36 @@ export const SearchApplicationIndicesFlyout: React.FC = () => {
     isFlyoutVisible,
   } = useValues(SearchApplicationIndicesFlyoutLogic);
   const { closeFlyout } = useActions(SearchApplicationIndicesFlyoutLogic);
-
+  const { navigateToUrl, share } = useValues(KibanaLogic);
+  const searchIndicesLocator = useMemo(
+    () => share?.url.locators.get('SEARCH_INDEX_DETAILS_LOCATOR_ID'),
+    [share]
+  );
+  const SearchIndicesLinkProps = useCallback(
```

Comment:
see other comment.

Reply:
updated in 6c03f45068133d97067aef5429bb53e1c4c97990